core/include/usListenerFunctors.h

```diff
@@ -34,6 +34,25 @@ namespace us {
   class BundleEvent;
 
   typedef std::function<void(const ServiceEvent&)> ServiceListener;
+
+  /**
+   * \ingroup MicroServices
+   *
+   * A \c BundleEvent listener. When a \c BundleEvent is fired, it is
```

Comment:
Now that BundleListener is asynchronous, we need implementation for SynchronousBundleListener to support extender pattern.